README.adoc

```diff
@@ -1164,6 +1164,32 @@ result = if some_condition; something else something_else end
 result = some_condition ? something : something_else
 ----
 
+=== Use `case` [[use-case]]
+
+Prefer `case` over `if-elsif`.
```

Comment:
Do you think it makes sense to mention that this only applies to the case when the value being compared is always the same?
I.e.
```
if foo == :foo
  foo_action
elsif bar == :bar
  bar_action
end
```
doesn't really apply, since can't be expressed nicely with `case`.

Reply:
Agreed.

Reply:
Fixed. Improved the example as well.